packages/aztec.js/src/proof/proofUtils.js

```diff
@@ -6,7 +6,7 @@
 
 const {
     errors: { customError },
-    constants: { errorTypes, K_MAX },
+    constants,
```

Comment:
We should be consistent on whether we destructure imports or not. In verifier.js for dividentComputation we breakout errorTypes, why not here?

Reply:
I agree, we should standardise across the whole repo. This is a fairly large standardisation job though, and I think out of scope for this PR. 

I think Paul's refactor will go a long way to standardising imports, so will leave this for now.